src/healthkit.cs

```diff
@@ -971,6 +999,11 @@ public interface HKQuery {
 		[Export ("predicateForWorkoutsWithOperatorType:totalDistance:")]
 		NSPredicate GetPredicateForTotalDistance (NSPredicateOperatorType operatorType, HKQuantity totalDistance);
 
+		[iOS (10,0), Watch (3,0)]
+		[Static]
+		[Export ("predicateForWorkoutsWithOperatorType:totalSwimmingStrokeCount:")]
+		NSPredicate GetPredicateForTotalSwimmingStrokeCount (NSPredicateOperatorType operatorType, HKQuantity totalSwimmingStrokeCount);
```

Comment:
I was not sure about the name.... but it does match the older ones (and it would be hard to get a name with the overloads). Consistency +1 :)